src/magma/vk/buffers/BufferManager.cpp

```diff
@@ -65,107 +62,104 @@ Buffer& BufferManager::createBuffer(const std::string &name, VkDeviceSize size,
     return getBuffer(name);
 }
 
-Buffer& BufferManager::createStagingBuffer(const std::string &name, VkDeviceSize size, VkBufferUsageFlags usage) {
-    return createBuffer(name, size, usage, VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT | VK_MEMORY_PROPERTY_HOST_COHERENT_BIT);
+Buffer& BufferManager::createStagingBuffer(const std::string &name, vk::DeviceSize size, vk::BufferUsageFlags usage) {
+    return createBuffer(name, size, usage, vk::MemoryPropertyFlagBits::eHostVisible | vk::MemoryPropertyFlagBits::eHostCoherent);
 }
 
-Buffer& BufferManager::createDeviceBuffer(const std::string &name, VkDeviceSize size, VkBufferUsageFlags usage) {
-    return createBuffer(name, size, usage, VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT);
+Buffer& BufferManager::createDeviceBuffer(const std::string &name, vk::DeviceSize size, vk::BufferUsageFlags usage) {
+    return createBuffer(name, size, usage, vk::MemoryPropertyFlagBits::eDeviceLocal);
 }
 
-Buffer& BufferManager::createUniformBuffer(const std::string &name, VkDeviceSize size) {
-    return createStagingBuffer(name, size, VK_BUFFER_USAGE_UNIFORM_BUFFER_BIT);
+Buffer& BufferManager::createUniformBuffer(const std::string &name, vk::DeviceSize size) {
+    return createStagingBuffer(name, size, vk::BufferUsageFlagBits::eUniformBuffer);
 }
 
-Buffer& BufferManager::createBufferWithData(const std::string &name, const void* data, VkDeviceSize dataSize, 
-        VkBufferUsageFlags usage, VkMemoryPropertyFlags properties) {
-    bool isDeviceBuffer = properties & VK_MEMORY_PROPERTY_DEVICE_LOCAL_BIT;
+Buffer& BufferManager::createBufferWithData(const std::string &name, const void* data, vk::DeviceSize dataSize, 
+        vk::BufferUsageFlags usage, vk::MemoryPropertyFlags properties) {
+    auto isDeviceBuffer = properties & vk::MemoryPropertyFlagBits::eDeviceLocal;
```

Comment:
isDeviceBuffer can be created with type bool.
But now it's vk::MemoryPropertyFlags (or smth)